docs/volcano-integration.md

```diff
@@ -0,0 +1,87 @@
+# Integrate with volcano scheduler for Spark Application
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workload.
+Now the spark application pods can be scheduled via volcano for better scheduling efficiency.
```

Comment:
I suggest rephrasing this to `With the integration with Volcano, Spark application pods can be scheduled ...`.